### DISCUSSION TITLE
Season titles

### DISCUSSION THREAD

**Discussion by Gaalen69** [rrirower/Channel-Feed-Manager#68] [Feature requests]
Would it be possible to add Season titles? I have my episodes listed by season on Roku and I would like the user to be able to jump to each season by clicking on a season title. Right now, I am adding that data point manually to the JSON feed file and it all works beautifully, however to my chagrin, when I modified the JSON file through the Channel Manager Interface, it wiped all the manually added data points. I can understand if this isn't a high priority, so no worries if its something that has to happen further down the road. I will just have to do everything manually in the mean time. Let me know if this is a possibility. Love the program, btw. Thanks!

**Comment by rrirower** [comment]
Currently, the software follows the Roku JSON feed spec for Direct Publisher.  It uses the Season number to identify each season.  It does not understand the concept of a Season title.  I will mark this as an enhancement, but, I don't have a timeframe for adding it.
